[opdocs] Kommander: Split clusters into multiple pages (#3400)
mesosphere/dcos-docs-site: pages/dkp/kommander/1.3/clusters/upgrading-kubernetes/index.md

---
layout: layout.pug
beta: true

> navigationTitle: Upgrading Kubernetes Versions

Reviewer: ```suggestion
navigationTitle: Upgrade Kubernetes version
```

> title: Upgrading Kubernetes Versions

Reviewer: ```suggestion
title: Upgrade Kubernetes version
```

menuWeight: 7
excerpt: Upgrade the Kubernetes version on a cluster
---

You can upgrade the Kubernetes version on a running cluster from the Cluster Detail page (Figure 2).

> Select an available desired version from the Version dropdown and confirm your action in the confirmation dialog.

Reviewer: ```suggestion
Select the specific version from the Version dropdown and confirm your action in the confirmation dialog.
```
